lib/event.ts

```diff
@@ -0,0 +1,28 @@
+import { LogDescription } from '@ethersproject/abi';
+import { ContractReceipt } from 'ethers';
+import { Interface } from 'ethers/lib/utils';
+
+export function getLogs(
+  contractInterface: Interface,
+  receipt: ContractReceipt,
+  only: string[] = [],
+) {
+  const events: LogDescription[] = [];
+
+  for (const log of receipt.logs) {
+    try {
+      const description = contractInterface.parseLog(log);
```

Comment:
I don't like this variable name.  Can change it in a future reorganization, though.

Reply:
renamed this to `parsedLog`